chain-core/src/tx/witness/mod.rs

```diff
@@ -51,6 +52,7 @@ impl ::std::ops::DerefMut for TxWitness {
 
 // normally should be some structure: e.g. indicate a type of signature
 #[derive(Debug, PartialEq, Eq, Clone)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
`Serialize` and `Deserialize` shouldn't be added here.

Reply:
i'll try  to compile again without "serde".